pkg/api/validation/schema.go

```diff
@@ -66,6 +66,19 @@ func NewSwaggerSchemaFromBytes(data []byte) (Schema, error) {
 	return schema, nil
 }
 
+// unpackObject returns obj's items element as an array of interface.
+// Returns an error if obj is not a List type (does not have an items member).
+func unpackObject(obj map[string]interface{}) ([]interface{}, error) {
+	items := reflect.ValueOf(obj).MapIndex(reflect.ValueOf("items"))
```

Comment:
Wait, this whole function is a convoluted way to do `list, ok := obj["items"].([]interface{})`

This function should not use reflection.